presto-main/src/main/java/com/facebook/presto/metadata/FunctionAndTypeManager.java

```diff
@@ -111,9 +113,9 @@
     private final BlockEncodingSerde blockEncodingSerde;
     private final BuiltInTypeAndFunctionNamespaceManager builtInTypeAndFunctionNamespaceManager;
     private final FunctionInvokerProvider functionInvokerProvider;
-    private final Map<String, FunctionNamespaceManagerFactory> functionNamespaceManagerFactories = new ConcurrentHashMap<>();
+    private final Map<String, FunctionNamespaceManagerFactory> functionNamespaceManagerFactories;
     private final HandleResolver handleResolver;
-    private final Map<String, FunctionNamespaceManager<? extends SqlFunction>> functionNamespaceManagers = new ConcurrentHashMap<>();
+    protected final Map<String, FunctionNamespaceManager<? extends SqlFunction>> functionNamespaceManagers;
```

Comment:
hmmmm do we need this change for the first commit? I didn't find any usage of these to be protected outside this class